packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -29,7 +29,7 @@ export interface IComboBoxState {
   isOpen?: boolean;
 
   // The currently selected index (-1 if no index is selected)
```

Comment:
@jspurlin, there appears to be some legitimate bugs in ComboBox exposed by enabling strictNullChecks. There appears to be some situations where we aren't handling null and undefined values correctly. These changes should be functionally equivalent to the previous combo box for the most part, but we should go back in here and address these issues by either being more strict in our types (not allowing null/undefined for some types, removing some optionals, or properly handling null/undefined)